context/on_startup.py

```diff
@@ -1,39 +1,26 @@
-from os import listdir
 import json
 import SimpleHTTPServer
 import SocketServer
-import re
-
-def as_track(file):
-    return {
-        "name": re.sub(r'.*\/', '', file),
-        "url": 'data/' + file,
-        "format": re.sub(r'.*\.', '', file)
-        # TODO: indexURL if BAM
-    }
 
-def populate_data_directory():
-    # files.txt is really for testing rather than production use.
-    files = '\n'.join(listdir('data'))
-    with open('data/files.txt', 'w') as list_file:
-        list_file.write(files)
 
-    # options.json is actually used by IGV.js.
-    # TODO: How does Refinery communicate the genome assembly?
+def populate_igv_configuration():
+    with open("data/input.json") as f:
+        config_data = json.loads(f.read())
+
+    tracks = []
     assembly = "hg19"
```

Comment:
This can be the next step, but Refinery needs to be able to specify the genome assembly at some point.